index.bs

```diff
@@ -13110,12 +13123,14 @@ The following typographic conventions are used in this document:
 The following conventions are used in the algorithms in this document:
 
 *   Algorithms use the [=conventions of the ECMAScript specification=],
-    including the ! and ? notation for unwrapping completion records.
-*   Algorithms sometimes treat returning/throwing values and returning completion records
+    including the ! and ? notation for unwrapping [=Completion Records=].
```

Comment:
Would have been nice to link directly to the Completion Record dfn, here, if you're so inclined to file a bug with the TC-39 spec.